src/jdk.compiler/share/classes/com/sun/tools/javac/resources/compiler_de.properties

```diff
@@ -403,6 +406,10 @@ compiler.err.duplicate.unconditional.pattern=Doppeltes nicht bedingtes Muster
 
 compiler.err.unconditional.pattern.and.default=Switch umfasst sowohl ein nicht bedingtes Muster als auch ein Standardlabel
 
+compiler.err.unconditional.pattern.and.both.boolean.values=Switch umfasst sowohl boolesche Werte als auch ein nicht bedingtes Muster
```

Comment:
Here and elsewhere in this file and other translations, I'm suprised `switch` is not being translated. In the English form, it is not being used as a keyword, although could maybe be seen as a short form of `switch' statement, in which case `switch` should be treated as a keyword and not translated or capitalized.

Reply:
Good catch. Seems to be across all files and I'm assuming it's for the reason you suggested. I'll bring this up to the go team. I have no way to fix this since I can't rely on google translate. This can be caught in a future update if the go team can successfully differentiate the difference between the word `switch` and a `switch keyword`.

Reply:
Yes. "Switch ..." is the short form for "Das Switch-Statement ..." or "Das `switch` Statement ..." and all other longer forms that match the context.